packages/fe-fpm-writer/test/unit/action.test.ts

```diff
@@ -275,6 +275,19 @@ describe('CustomAction', () => {
                 ).toMatchSnapshot();
             });
 
+            test('"eventHandler" is "object", action with lowercase first letter', () => {
+                generateCustomActionWithEventHandler(name, {
+                    fnName: 'dummyOnAction'
+                });
+
+                const manifest = fs.readJSON(join(testDir, 'webapp/manifest.json'));
+                const action = getActionByName(manifest, name);
+                expect(action['press']).toEqual('my.test.App.ext.myCustomAction.MyCustomAction.dummyOnAction');
+                expect(
```

Comment:
To we need to store a snapshot here? Isn't the purpose of the test to check the existence of the file at the correct path?